packages/langchain/src/openai/util.test.ts

```diff
@@ -74,7 +74,8 @@ describe('Mapping Functions', () => {
       ...defaultOptions,
       ...options
     });
-    expect(mapping).toMatchObject(request);
+    // Todo find a better way to remove undefined properties
+    expect(JSON.parse(JSON.stringify(mapping))).toStrictEqual(request);
```

Comment:
Instead of changing the expect statement, just add another  one:

expect(mapping.messages.filter(message => message.role === "assistant")[0].tool_calls).toBeNotDefined()